src/components/Loading.jsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import {Container, Row, Card} from 'react-bootstrap';
 
-const Loading = () => {
+const Loading = ({data}) => {
```

Comment:
I see what you're doing here. But `data` is a misleading variable name, it implies there is data being passed in and not just a string.

Maybe instead pass in a prop called `loadingMessage` with a default of `"Loading"`?